ArkKit/ark-state-kit/ark-ecs-kit/InternalSystems/ArkTimeSystem.swift

```diff
@@ -0,0 +1,23 @@
+import Foundation
+
+/**
+ *  A system which updates a StopWatchComponent instances in ArkECS after the given delta.
+ */
+class ArkTimeSystem: System {
+    var active: Bool
+    
+    init(active: Bool = true) {
+        self.active = active
+    }
+    
+    func update(deltaTime: TimeInterval, arkECS: ArkECS) {
+        let stopWatchEntities = arkECS.getEntities(with: [StopWatchComponent.self])
+        for stopWatchEntity in stopWatchEntities {
+            guard let stopWatchComponent = arkECS.getComponent(ofType: StopWatchComponent.self, for: stopWatchEntity) else {
+                return
```

Comment:
i think this should be `continue` instead. If i am not wrong, using `return` would cause an exit to the function.